hackathon_site/hardware/views.py

```diff
@@ -112,6 +114,19 @@ class IncidentDetailView(mixins.RetrieveModelMixin, generics.GenericAPIView):
     def get(self, request, *args, **kwargs):
         return self.retrieve(request, *args, **kwargs)
 
+    def patch(self, request, *args, **kwargs):
+        try:
+            data = request.data
+            if not isinstance(data, dict):
+                raise ValueError("Invalid request data format")
+            valid_fields = {"state", "time_occurred", "description", "order_item"}
```

Comment:
If I am understanding your code properly, I think removing "order_item" from this line (line 122) should resolve the issue. If this isn't the case feel free to implement your own strategy to resolve the issue with order_item!